tests/util_compare_json_reports.py

```diff
@@ -0,0 +1,88 @@
+# Copyright contributors to the oneDAL project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import json
+import sys
+
+
+def load_json_from_file(fname: str) -> dict:
+    with open(fname, "r") as f:
+        return json.load(f)
+
+
+def dct_wo_key(dct: dict, k_remove: str) -> dict:
+    return {k: v for k, v in dct.items() if k != k_remove}
```

Comment:
```suggestion
def dct_wo_key(dct: dict, k_remove: str) -> dict:
      return (dct.pop('a', None), dct)[1]
```
? (If you wanted a one-liner, outside of that I would do it in two for simplicity)

Reply:
That would modify the dict in-place, which can be problematic given that it's used in other functions.